monero-outreach-docs/translations/fr/archive/rpc_pay_fr.md

```diff
@@ -0,0 +1,133 @@
+
+### _RPC_
+
+Avant d'approfondir le fonctionnement du Paiement-RPC, il est opportun de considérer le concept général derrière le fonctionnement des Appels de Procédures à Distance (RPC). Il s'agit d'un processus par lequel un programme (le client) effectue une requête pour des données et des calculs venant d'un autre programme (le serveur). Le serveur peut être situé sur n'importe quel ordinateur sur l'Internet. Une requête RPC s'apparente à un appel de procédure imbriqué dans un program régulier, à l'exception qu'avec RPC le travail de la procédure est effectué sur le serveur. Un logiciel intermédiaire dédié traite la communication de la requête et la réponse sur le réseau afin que le tout se fasse de manière fluide. Plusieurs logiciels ont été développés pour servir d'interface et permettre l'implémentation de RPC au fil des ans.
+
+Une itération plus récente de logiciels d'interface RPC fait appel au langage de JavaScript Object Notation (JSON), un format textuel standard pour l'échange de données, pour la formation des messages. Grâce à des appariements de noms et valeurs ainsi que des listes de données, la définition des informations est robuste, extensible et lisible à l'humain. L'encodage JSON est désormais la méthode principale utilisée par l'infrastructure RPC de Monero.
+
+Pour Monero, le processus RPC est déjà utilisé à plusieurs fins. Par exemple, le démon standard de Monero, qui contient la plupart des fonctionnalités de Monero, est _monerod_. Il implémente le RPC côté serveur pour plusieurs appels de fonctions reliés aux fonctionnalités de la chaîne de blocs. Le programme _monero-wallet-rpc_ (inclus avec l'installation standard de Monero), implémente l'instance côté serveur de la fonctionnalité du portefeuille tout en étant lui-même le client d'un serveur _monerod_. Les appels RPC de Monero opèrent d'une manière similaire à celle d'un fureteur internet qui communique avec un serveur web. Les données sont envoyées avec la méthode POST, et l'information retournée est textuelle (habituellement en JSON). Si vous voulez des exemples détaillés de messages RPC, voyez [Exemple #1](https://www.monerooutreach.org/stories/RPC-Pay.html#box1) sur l'utilisation des appels RPC Monero.
```

Comment:
fureteur >>> navigateur   

Reason: Just a suggestion. 'fureteur' is I think specific to French-Canadian (I heard it only in very rare occasions)

Reply:
Possibly a French-Canadian thing. Navitageur works too, so I integrated it.